### PR TITLE
Add structured geocoding docs

### PR DESCRIPTION
**Pull request by rmglennon**
Adds new topic for structured (needs final blog content copied in, as this is kind of a page dump)
Adds some content into search (although it feels kind of bolted on, as all the examples are of text parameter

Depends on https://github.com/mapzen/mapzen-docs-generator/pull/236

### CODE REVIEW

File: search.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ All Mapzen Search requests share the same format:
   scheme       domain   version  path     query     authentication token
 ```
 
+In addition, you can search at `https://search.mapzen.com/v1/search/search/structured` to look for individual components of a location.
```
Comment (trescube) [inline search.md]: too many `/search`'s

**Comment by rmglennon** [comment]
OK, these can be reviewed now.

The anchor link to  search.md/#available-search-parameters doesn't seem to be moving to the right location, so I can investigate.

When this PR is in master, we can merge the docs generator pull request and get these live (if even as a placeholder and make edits later).

File: structured-geocoding.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
+# Structured geocoding
+
+With structured geocoding, you can search for the individual parts of a location. Structured geocoding is an option on the [`search` endpoint](search.md), where a query takes the form of https://search.mapzen.com/v1/search/structured.
+
+For example, you want to find `30 West 26th Street, New York, NY`. With the geocoding parameter for `search`, you can only enter the entire location as one string, such as `text=30 West 26th Street, New York, NY`. However, with `search\structured`, you can specify that this location is composed of a street address, a locality, and a region.
```
Comment (trescube) [inline structured-geocoding.md]: `\` -> `/`

File: structured-geocoding.md [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+
+Note that the other [`search` parameters](search.md/#available-search-parameters) can also be combined with these, allowing you to filter and prioritize your results.
+
+### address
+
+The `address` parameter can contain a full address with house number or only a street name.
+
+_Examples_
+
+* [201 Spear Street](http://search.mapzen.com/v1/search/structured?address=201%20Spear%20Street&locality=San%20Francisco&region=CA)
```
Comment (trescube) [inline structured-geocoding.md]: It's really annoying, but for some reason clicking a URL containing spaces or `%20`s doesn't work, they have to be `+`s.

File: structured-geocoding.md (hunk 2 above)
Comment (rmglennon) [inline structured-geocoding.md]: D'oh. good catch

**Comment by rmglennon** [comment]
OK, changed the / to be forward, and also updated the links with spaces to use + from the blog (after we discovered the cache/flush problem).

**Pull request merged**